README.md

```diff
@@ -0,0 +1,22 @@
+Pipeline: Nodes and Processes
+===
+
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/workflow-durable-task-step)](https://plugins.jenkins.io/workflow-durable-task-step)
+[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/workflow-durable-task-step-plugin?label=changelog)](https://github.com/jenkinsci/workflow-durable-task-step-plugin/blob/master/CHANGELOG.md)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/workflow-durable-task-step?color=blue)](https://plugins.jenkins.io/workflow-durable-task-step)
+
+A component of [Pipeline Plugin](https://wiki.jenkins.io/display/JENKINS/Pipeline+Plugin).
+
+Pipeline steps locking agents and workspaces, and running external processes that may survive a Jenkins restart or agent reconnection.
+
+
+>**WARNING**: Older versions of this plugin may not be safe to use. Please review the following warnings before using an older version:
```

Comment:
This should be removed, the plugin site shows equivalent content in the sidebar to the right.

Reply:
Good catch, see https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/124.